src/mlpack/methods/cf/cf_main.cpp

```diff
@@ -364,16 +375,20 @@ void PerformAction(CFModel* c)
   CLI::GetParam<CFModel*>("output_model") = c;
 }
 
-template<typename DecompositionPolicy>
+template <typename DecompositionPolicy>
```

Comment:
I think most of the times, we write `template<typename>`, so I think we don't have to change it.